api/types/src/derives.rs

```diff
@@ -36,7 +36,14 @@ impl_poem_type!(
             "0x88fbd33f54e1126269769780feb24480428179f552e2313fbe571b72e62a1ca1 ".to_string()
         )),
         format = Some("hex"),
-        description = Some("Hex encoded 32 byte Aptos account address")
+        description = Some(indoc! {"
+            The address of an account
```

Comment:
I'd still keep this line as `Hex encoded 32 byte Aptos account address`, to make it clear that it's a 32 byte address. Otherwise I'm confident people will get confused by "64 character hex string".